### ISSUE TITLE
Vulnerability detected in a particular version in build.gradle

### ISSUE THREAD

**Issue by chughpiyush** [crate/crate-java-testing#64]
Lately I was testing for vulnerabilities in my application and it was when I realized that one of the transitive vulnerable dependency was coming in from this project.

Then I ran Owasp dependency checker and got to know that `commons-compress-1.2.jar` is vulnerable to [CVE-2012-2098](https://nvd.nist.gov/vuln/detail/CVE-2012-2098).

```
CVE-2012-2098  suppress

Algorithmic complexity vulnerability in the sorting algorithms in bzip2 compressing stream (BZip2CompressorOutputStream) in Apache Commons Compress before 1.4.1 allows remote attackers to cause a denial of service (CPU consumption) via a file with many repeating inputs.
CWE-310 Cryptographic Issues
```

I think we should fix it in this project and I can give a Pull Request as soon as this issue is approved.

![image](https://user-images.githubusercontent.com/30496180/73551254-0e94a200-446c-11ea-94f6-be603bb3f257.png)

**Comment by mfussenegger** [comment]
:+1:

A PR that updates the dependencies would be appreciated.

**Comment by chughpiyush** [comment]
> A PR that updates the dependencies would be appreciated.

I would need to sign the CLA first, is that right?

**Comment by mfussenegger** [comment]
> 
> 
> > A PR that updates the dependencies would be appreciated.
> 
> I would need to sign the CLA first, is that right?

yes

**Comment by seut** [comment]
This issue is solved by https://github.com/crate/crate-java-testing/pull/67.

@chughpiyush Thank you again for raising this and contributing a fix.

**Comment by seut** [comment]
@chughpiyush We've published v0.8.0 including your contributions.
